git/merge-a-branch-as-a-single-commit.md

```diff
@@ -0,0 +1,12 @@
+# Merge all changes from another branch as a single commit:
+
+Let's say you want to merge "my-feature" (which contain 10 commits :D ) into develop:
+
+    git checkout develop
+    git merge --squash my-feature
```

Comment:
Hello zamith, As someone who getting started with git I found git merge --squash simpler for this use case. 

If you'd like you can add an exemple of rebase -i as another way to do it.

Thanks